parsers/powershell.c

```diff
@@ -72,8 +71,24 @@ typedef enum eTokenType {
 	TOKEN_VARIABLE
 } tokenType;
 
+enum {
+	KEYWORD_function,
+	KEYWORD_filter,
```

Comment:
If you choose "unifying-function-and-filter approach", KEYWORD_filter is not needed.
About "unifying-function-and-filter approach", see below.

Reply:
We chose the "introducing-a-kind-for-filter approach". For more information, see https://github.com/universal-ctags/ctags/pull/3480#discussion_r977902309.